Makefile

```diff
@@ -1,7 +1,9 @@
-all: tests.js
+all: test
 
-tests.js: FORCE $(shell find src test -type f -name '*.elm' -o -name '*.js')
-	elm-make --yes --warn
-	@$(MAKE) -C test
+install: package.json
+	npm install
+
+test: install
+	./node_modules/.bin/elm-test
```

Comment:
@krisajenkins  not really sure on your preferences to how you want to do this :)